### DISCUSSION TITLE
ZSH support: Testers needed!

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [mylinuxforwork/dotfiles#143] [Announcements]
The ML4W Dotfiles Rolling Release are now including an option to switch to zsh. This is a feature requested form many users.

Please check the following Wiki article for more information:
https://github.com/mylinuxforwork/dotfiles/wiki/Switch-between-bash-and-zsh

Please feel free to install the latest rolling release and share your feedback and additional requests here.

**Comment by dwilliam62** [comment]
I updated to 2.96RL   Changed shell to ZSH via APP    oh-my-zsh installed.  I rebooted and I get this when I start terminal 

![image](https://github.com/user-attachments/assets/4d718975-a98d-4db0-8078-d3ee4a80e5bc)

**Reply by mylinuxforwork** [reply]
You were super fast. I have just pushed an update to install oh-my-posh with yay. Just install it with yay -S oh-my-posh

**Comment by dwilliam62** [comment]
![image](https://github.com/user-attachments/assets/a14cc66c-6fd4-40ab-a2d8-e09e5edb286b)

**Comment by dwilliam62** [comment]
![image](https://github.com/user-attachments/assets/72833ff8-cc60-4d82-915b-95e30c6a1834)

**Comment by dwilliam62** [comment]
I installed oh-my-posh

**Comment by dwilliam62** [comment]
I did on another arch VM and it worked fine.    FYI:  I am filing two more issues.  1.)  In the automation.sh I have neovim set to false AND hook.sh to protect .config/nvim and every upgrade since setting it up overwrite my NVIM config  2.)  I thought before you fixed it so wallpaper wasn't reset to default

**Reply by mylinuxforwork** [reply]
I will check.

**Comment by faxxxmaster** [comment]
The alias "ll" comes from the file ~/.oh-my-zsh/lib/directories.zsh
![image](https://github.com/user-attachments/assets/ecdcb525-ef4b-4ebe-89d4-bdb139ebb914)
After commenting out, everything works correctly

**Reply by hambosto** [reply]
but alias "ls" for 'eza -a --icons' still not working. i don't know where the alias for that is ls.

![screenshot_17082024_144125](https://github.com/user-attachments/assets/2f9e2def-a0ea-4ed1-9465-a2c43e57b739)

**Reply by faxxxmaster** [reply]
I found these additional files where aliases for ls are defined

![image](https://github.com/user-attachments/assets/e8321645-c0c6-45f7-a484-d107bff881c1)

**Reply by hambosto** [reply]
i see. it definitely override our custom zsh configuration aliases. i hope it will fixed soon in the next version @mylinuxforwork

**Reply by mylinuxforwork** [reply]
You need to transfer the aliases into an additional file in .config/zshrc/.

Please see https://github.com/mylinuxforwork/dotfiles/wiki/Switch-between-bash-and-zsh

**Comment by Homer-Max** [comment]
/home/kyle/.config/zshrc/20-customization:source:27: no such file or directory: /home/kyle/.oh-my-zsh/oh-my-zsh.sh
I lack this file

**Reply by dwilliam62** [reply]
You can manually install oh-my-zsh and oh-my-posh to resolve this.  yay -S oh-my-posh oh-my-zsh
